src/loginPage/loginPage.component.tsx

```diff
@@ -416,6 +426,7 @@ const LoginPageComponent = (props: CombinedLoginProps): React.ReactElement => {
   const authUrl = props.auth.provider.authUrl;
   const [mnemonics, setMnemonics] = useState<ICATAuthenticator[]>([]);
   const [fetchedMnemonics, setFetchedMnemonics] = useState<boolean>(false);
+  const [isComponentVisible, setComponentVisible] = useState<boolean>(false);
```

Comment:
Slight comment that could this be renamed to be something like `loading` and `setLoading` - you'd have to reverse the logic (or use `loaded` and `setLoaded` I suppose...) - `isComponentVisisble` is a bit confusing